src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/src/api-authorization/logout/logout.component.ts

```diff
@@ -0,0 +1,117 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
There's a bunch of TSLint errors in this file.

Reply:
Actually all to do with quotes I think.

Reply:
Done